src/ethereum_test_specs/blockchain.py

```diff
@@ -300,6 +300,7 @@ class BlockchainTest(BaseTest):
     genesis_environment: Environment = Field(default_factory=Environment)
     verify_sync: bool = False
     chain_id: int = 1
+    exclude_post_state: bool = False
```

Comment:
This is a new flag to signal that we want to output an empty `postState` in the fixture, to avoid the mentioned bloat.

This is an easy proposal, I would prefer experts in the testing team to asses if this is okay or is another prefered option. :)

Reply:
I think this alone should be a separate PR to have a better discussion on it, could you create another PR independent just for this pls?

Reply:
Sure, I'll do that.

Reply:
@marioevz, created it here https://github.com/ethereum/execution-spec-tests/pull/1578.
After we see how that goes, I can rebase this PR.